packages/sirius-web/frontend/sirius-web-application/src/index.ts

```diff
@@ -74,6 +74,10 @@ export { type ProjectActionButtonProps } from './views/project-browser/list-proj
 export { projectActionButtonMenuItemExtensionPoint } from './views/project-browser/list-projects-area/ProjectActionButtonExtensionPoints';
 export { type ProjectRowProps } from './views/project-browser/list-projects-area/ProjectRow.types';
 export { projectsTableRowExtensionPoint } from './views/project-browser/list-projects-area/ProjectsTableExtensionPoints';
+export { type ProjectImagesSettingsModal } from './views/project-settings/images/ProjectImagesSettings.types';
+export { UploadImageModal } from './views/project-settings/images/upload-image/UploadImageModal';
+export { useProjectImages } from './views/project-settings/images/useProjectImages';
+export { type GQLImageMetadata } from './views/project-settings/images/useProjectImages.types';
```

Comment:
I don't mind exporting those components and types but keep in mind that they are built for our internal consumption first so they may change in the future. It does not seem likely that they will in the near future because I don't think we have any planned improvements on this part of the code soon but just keep that in mind.